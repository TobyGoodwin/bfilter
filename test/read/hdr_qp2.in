Subject: =?utf-8?q?=E2=9C=89_New_message_from_ncrbrts?=

this is the body
