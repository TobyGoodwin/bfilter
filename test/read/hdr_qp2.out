{ âœ‰ New message from ncrbrts}
{this is the body}
